### PR TITLE
passed all tests

### PR DESCRIPTION
**Pull request by cuius21**

### CODE REVIEW

File: app/src/main/java/pl/edu/pk/mobile/tourtool/service/model/User.kt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package pl.edu.pk.mobile.tourtool.service.model
 
 data class User(
-  val firstName: Name,
-  val lastName: Name,
-  val email: Email,
-  val password: Password
+  val firstName: String,
```
Comment (M1ST4KE) [inline app/src/main/java/pl/edu/pk/mobile/tourtool/service/model/User.kt]: WHY?

**Review by M1ST4KE — CHANGES_REQUESTED** [review]

File: app/src/main/java/pl/edu/pk/mobile/tourtool/service/model/User.kt (hunk 1 above)
Comment (cuius21) [reply app/src/main/java/pl/edu/pk/mobile/tourtool/service/model/User.kt]: ok, fixed

File: app/src/main/java/pl/edu/pk/mobile/tourtool/service/dto/UserDTO.kt [hunk 2]
```diff
@@ -2,20 +2,23 @@ package pl.edu.pk.mobile.tourtool.service.dto
...
 import com.google.gson.annotations.SerializedName
+import pl.edu.pk.mobile.tourtool.service.model.Email
+import pl.edu.pk.mobile.tourtool.service.model.Name
+import pl.edu.pk.mobile.tourtool.service.model.Password
 
 data class UserDTO(
   @SerializedName("firstName")
   @Expose
-  val firstName: String,
+  val firstName: Name,
```
Comment (M1ST4KE) [inline app/src/main/java/pl/edu/pk/mobile/tourtool/service/dto/UserDTO.kt]: It has to be `String`

**Review by M1ST4KE — CHANGES_REQUESTED** [review]

File: app/src/main/java/pl/edu/pk/mobile/tourtool/service/dto/UserDTO.kt (hunk 2 above)
Comment (cuius21) [reply app/src/main/java/pl/edu/pk/mobile/tourtool/service/dto/UserDTO.kt]: ok

**Review by M1ST4KE — APPROVED** [review]

**Pull request merged**